compute_endpoint/globus_compute_endpoint/engines/globus_compute.py

```diff
@@ -217,32 +227,21 @@ def containerized_launch_cmd(self) -> str:
         launch_cmd = self.executor.launch_cmd
         # Adding assert here since mypy can't figure out launch_cmd's type
         assert launch_cmd
-        if self.container_type == "docker":
+        if self.container_type in ["docker", "podman", "podman-hpc"]:
```

Comment:
As mentioned above, this could use an already-defined structure, rather than being a second list that developers must (remember to) maintain.